pibooth/booth.py

```diff
@@ -282,6 +282,13 @@ def find_capture_event(self, events):
                 rect = self._window.get_rect()
                 if pygame.Rect(0, 0, rect.width // 2, rect.height).collidepoint(event.pos):
                     return event
+            if event.type == pygame.FINGERDOWN:
+                LOGGER.debug("Touch action dectected: %s", event)
+                display_size = pygame.display.get_surface().get_size()
```

Comment:
use self._window.display_size

(in pygame<2, display size can be get only before first window is opened, else the window size is returned)

Reply:
Not sure to understand the reason but I agree it is better to use the window display size instead of getting it every time.